### ISSUE TITLE
Disable no command

### ISSUE THREAD

**Issue by HumanTree92** [ESX-Org/esx_rpchat#23]
So i have had some new people to FiveM join our server & instead of them doing /ooc or /me or any of the commands they just don't put any of that. Is there a way to where if someone doesn't put /ooc it gets sent to proximity players or tells you to please do /ooc.

**Comment by HumanTree92** [comment]
Just a little example on what i mean. Normally if someone wanted to talk out of character they would do "/ooc Message Here" but we have lots of players ignoring this & their just doing "Message Here".

**Comment by nearbyplayer** [comment]
That's their problem. If they're too stupid to follow a simple command-based system they shouldn't be on a roleplay server.

**Comment by nearbyplayer** [comment]
If you'd like to make a PR with what you've suggested by all means go for it.

**Comment by ElPumpo** [comment]
I agree with Human here. Maybe by default we should consider all messages ass ooc?
